src/override/functions.js

```diff
@@ -0,0 +1,63 @@
+export const getResource = (resources, resourceName) =>
```

Comment:
However I'm not a fond of the `override` name.

Can't we just move this file in file called `docUtils` in the root?

Reply:
Done